KD_Lib/original/mnist.py

```diff
@@ -6,47 +6,51 @@
 from .train import train_teacher, train_student
 from .evaluate import eval
 from .model import teacher, student
+from KD_Lib.RKD import RKDLoss
 
 
-def mnist(teacher_size=1200, student_size=800,epochs=20, lr=0.01, optimizer='SGD',
-          loss='MSE', batch_size=100, distil_weight=0.7, temp=20 ):
-    
+def mnist(teacher_size=1200, student_size=800, epochs=20, lr=0.01,
+          optimizer='SGD', loss='MSE', batch_size=100, distil_weight=0.7,
+          temp=20, rkd_angle=0.4, rkd_dist=0.6):
+
```

Comment:
add doc string here so people know about various options

Reply:
We could do that now or later when we formalize it into proper functions? If you still suggest, I will.

Reply:
yea do it now only, we will update it along the way

Reply:
:+1: